styles/alternativeto/catppuccin.user.css

```diff
@@ -0,0 +1,307 @@
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    --mainBrand: if(
+      @lookup = latte,
+      lighten(@accent-color, 10%),
+      darken(@accent-color, 50%)
+    );
+    --topBrand: darken(@accent-color, 10%);
+    --footer: darken(@accent-color, 50%);
+    --mainBg: @base;
+    --mainFg: @text;
+    --popBrand: @accent-color;
+    --introBoxText: @text;
+    --introBoxLink: @text;
+    --introBoxMeta: @subtext0;
+    --meta: @subtext0;
+    --metaLight: @subtext1;
+    --linkColorHeader: @text;
+    --linkColor: @text;
+    --brandLight3: @surface2;
+    --brandLight4: @surface2;
+    --brandLight5: @surface1;
+    --brandLight6: @mantle;
+    --brandLight7: @surface0;
+    --brandLight8: @text;
+    --brandLight10: @surface0;
+    --brandLight11: @mantle;
+    --gray50: @surface0;
+    --gray100: @surface0;
+    --gray150: @surface0;
+    --gray200: @surface1;
+    --gray250: @surface1;
+    --gray300: @surface1;
+    --gray500: @overlay0;
+    --headingButton: @accent-color;
+    --positiveGreener: @green;
+    --positiveGreenerLight: fade(@green, 10%);
+    --positiveGreenerDark: @green;
+    --danger: @red;
+    --negativeLight: fade(@peach, 10%);
+    --lightDanger: fade(@red, 25%);
+    --dangerDark: @red;
+    --ctaMain: @mauve;
+    --ctaLight: @base;
+    --ctaLightest: @mantle;
+    --tab: @mantle;
+    --toastify-color-dark: @mantle;
+    --toastify-color-light: @text;
+    --toastify-color-info: @blue;
+    --toastify-color-success: @green;
+    --toastify-color-warning: @yellow;
+    --toastify-color-error: @red;
+
+    // heart icon
+    [class^="ModernLikeButton_likeWrapper"] {
+      [class^="ModernLikeButton_heart"]:hover,
+      [class^="ModernLikeButton_liked"] {
+        svg {
+          color: @red;
+        }
+      }
+    }
+    // primary button
+    [class*="Button_primary"] {
+      color: @crust !important;
+    }
+    // footer
+    [class^="Footer_footer"] {
+      &,
+      & p a,
+      & [class^="Footer_footerMenu"] li a {
+        color: lighten(@accent-color, 10%);
```

Comment:
```suggestion
        color: @text;
```

Reply:
oops i accidentally merged this without testing it. this breaks in light mode
![image](https://github.com/user-attachments/assets/cf4478b3-46d9-4ad9-afff-bd7e6d060220)
i will revert it to
![image](https://github.com/user-attachments/assets/06848f52-d549-4550-8d34-16025452c350)
unless you want it to be text color normally and crust on light mode, i will keep it a lighter accent color. because this is closer to how the unthemed website looks like
![image](https://github.com/user-attachments/assets/d17e348e-6a0e-4df6-a00e-97dc43ff6571)